labhelper/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ target_include_directories( labhelper
 target_link_libraries ( labhelper
     PUBLIC
     imgui
-	SDL2::SDL2
```

Comment:
Could you comment this back in?

Reply:
Ah, brukar försöka att inte committa denna fil :(